components/dashboard/src/menu/Menu.tsx

```diff
@@ -128,52 +108,146 @@ export default function Menu() {
-                                        href: "https://www.gitpod.io/docs/",
-                                        target: "_blank",
-                                        rel: "noreferrer",
-                                    },
-                                    {
-                                        title: "Help",
-                                        href: "https://www.gitpod.io/support/",
-                                        target: "_blank",
-                                        rel: "noreferrer",
-                                        separator: true,
-                                    },
-                                    {
-                                        title: "Logout",
-                                        href: gitpodHostUrl.asApiLogout().toString(),
-                                    },
-                                ]}
-                            >
-                                <img
-                                    className="rounded-full w-6 h-6"
-                                    src={user?.avatarUrl || ""}
-                                    alt={user?.name || "Anonymous"}
-                                />
-                            </ContextMenu>
-                        </div>
+                        {/* Hide normal user menu on small screens */}
+                        <UserMenu user={user} className="hidden md:block" />
+                        {/* Show a user menu w/ admin & feedback links on small screens */}
+                        <UserMenu
+                            user={user}
+                            className="md:hidden"
+                            withAdminLink
+                            withFeedbackLink
+                            onFeedback={handleFeedbackFormClick}
+                        />
                     </div>
                     {isFeedbackFormVisible && <FeedbackFormModal onClose={onFeedbackFormClose} />}
                 </div>
             </header>
             <Separator />
+            {/* only shown on small screens */}
+            <OrgPagesNav className="md:hidden app-container flex justify-start py-2" />
+            {/* only shown on small screens */}
+            <Separator className="md:hidden" />
         </>
     );
 }
+
+type OrgPagesNavProps = {
+    className?: string;
+};
+const OrgPagesNav: FC<OrgPagesNavProps> = ({ className }) => {
+    const location = useLocation();
+
+    const leftMenu: Entry[] = useMemo(
+        () => [
+            {
+                title: "Workspaces",
+                link: "/workspaces",
+                alternatives: ["/"],
+            },
+            {
+                title: "Projects",
+                link: `/projects`,
+                alternatives: [] as string[],
+            },
+        ],
+        [],
+    );
+
+    return (
+        <div
+            className={classNames(
+                "text-base text-gray-500 dark:text-gray-400 flex items-center space-x-1 py-1",
```

Comment:
**suggestion**: The y-padding here is probably not needed as it's ignored by thy `py-2`, but it should be fine to resolve in a follow up PR to unblock this.

```suggestion
                "text-base text-gray-500 dark:text-gray-400 flex items-center space-x-1",
```